antarest/study/business/binding_constraint_management.py

```diff
@@ -111,9 +112,19 @@ def generate_id(self) -> str:
         return self.data.generate_id()
 
 
-class UpdateBindingConstProps(BaseModel):
-    key: str
-    value: Any
+class BindingConstraintForm(BaseModel, metaclass=AllOptionalMetaclass):
+    group: str
+    enabled: bool
+    time_step: BindingConstraintFrequency
+    operator: BindingConstraintOperator
+    filter_year_by_year: str
+    filter_synthesis: str
+    comments: str
+    coeffs: Dict[str, List[float]]
+
+
+class BindingConstraintEdition(BindingConstraintMatrices, BindingConstraintForm):
+    pass
```

Comment:
you can simplify to:

```python
class BindingConstraint(BindingConstraintConfig, metaclass=AllOptionalMetaclass):
    group: str
    coeffs: Dict[str, List[float]]

class BindingConstraintEdition(BindingConstraintMatrices, BindingConstraint):
    pass
```

Reply:
BindingConstraintConfig contains id and name so i don't think we should do that.

Reply:
mybad I mean `BindingConstraintProperties` not Config you're right it contains the id and name that should not be updated

Reply:
Ah okay. Why not, i'll let Laurent change this if he wants.

Reply:
We can also add camelCase aliases